assets/js/scenes/gameScene.js

```diff
@@ -252,6 +237,20 @@ const createGameScene = () => {
       layer("ui"),
       scale(0.4),
     ]);
+
+    // Quit Game
+    onKeyDown("escape", () => {
+      mainMusic.stop();
+      go("welcome");
```

Comment:
![Screenshot 2023-08-20 at 09 58 55](https://github.com/stephendawsondev/j-day/assets/57060620/5ca4e832-556d-4316-9b94-544b30bce81a)

Reply:
Yep! 
![image](https://github.com/stephendawsondev/j-day/assets/111069103/1207076f-883f-40c0-a51d-4548dcbe9b39)

Reply:
Not sure why that wasn't showing before, but it is now! Thanks!